src/DynamoCoreWpf/ViewModels/Menu/PreferencesViewModel.cs

```diff
@@ -696,6 +702,21 @@ public PreferencesViewModel(DynamoViewModel dynamoViewModel)
             preferencesTabs.Add("General", new TabSettings() { Name = "General", ExpanderActive = string.Empty });
             preferencesTabs.Add("Features",new TabSettings() { Name = "Features", ExpanderActive = string.Empty });
             preferencesTabs.Add("VisualSettings",new TabSettings() { Name = "VisualSettings", ExpanderActive = string.Empty });
+            preferencesTabs.Add("Package Manager", new TabSettings() { Name = "Package Manager", ExpanderActive = string.Empty });
+
+            //create a packagePathsViewModel we'll use to interact with the package search paths list.
+            var loadPackagesParams = new LoadPackageParams
+            {
+                Preferences = preferenceSettings,
+                PathManager = dynamoViewModel.Model.PathManager,
+            };
+            var customNodeManager = dynamoViewModel.Model.CustomNodeManager;
+            var packageLoader = dynamoViewModel.Model.GetPackageManagerExtension()?.PackageLoader;
+            var viewModel = new PackagePathViewModel(packageLoader, loadPackagesParams, customNodeManager);
+            
+            PackagePathsViewModel = new PackagePathViewModel(packageLoader, loadPackagesParams, customNodeManager);
```

Comment:
Why do we need two `packagepathviewmodel`s? Is `viewModel` in line 715 being used somewhere?

Reply:
@aparajit-pratap thank you for catching this, no it's not used, I'll remove that line.